.circleci/src/jobs/notify_release.yml

```diff
@@ -0,0 +1,25 @@
+# Create a release (with release notes) on github and send a mattermost notification.
+parameters:
+  production_delivery:
+    description: Whether builds are delivered to the production or beta lane of the play store.
+    type: boolean
+docker:
+  - image: cimg/node:19.1.0
+resource_class: small
+steps:
+  - prepare_workspace
+  - restore_environment_variables
+  - install_app_toolbelt
+  - run:
+      name: Create github release
+      command: echo "export RELEASE_ID='$(app-toolbelt v0 release create all ${NEW_VERSION_NAME} ${NEW_VERSION_CODE} --deliverino-private-key ${DELIVERINO_PRIVATE_KEY} --owner ${CIRCLE_PROJECT_USERNAME} --repo ${CIRCLE_PROJECT_REPONAME} --production-release << parameters.production_delivery >> --should-use-predefined-release-notes false)'" >> ${BASH_ENV}
+  - run:
+      name: Upload android apks to github release
+      command: app-toolbelt v0 release upload --releaseId ${RELEASE_ID} --files "$(ls ~/attached_workspace/*.apk)" --deliverino-private-key ${DELIVERINO_PRIVATE_KEY} --owner ${CIRCLE_PROJECT_USERNAME} --repo ${CIRCLE_PROJECT_REPONAME}
+  - run:
+      name: Upload ios ipas to github release
```

Comment:
not sure if we really need the ipas

Reply:
probably not really needed. I think we can delete it, to save tokens, but not really relevant though